### PR TITLE
Added changes for installation on Ubuntu.

### PR DESCRIPTION
**Pull request by oguzy**

### CODE REVIEW

**Comment by ywang19** [comment]
could you resolve the conflictions?

**Comment by jcalcote** [comment]
Hi Oguzy - your pull request has conflicts with master. Would you mind fetching and updating your master to the vsm master and then merging back into your branch? Then you can submit a pull request for this branch that has not conflicts with the current contents of vsm master.

**Comment by oguzy** [comment]
Done.

File: install.sh [hunk 1]
```diff
@@ -449,6 +449,7 @@ else
         exit 1
     else
 	sudo patch /usr/local/bin/vsm-controller < vsm-controller.patch
```
Comment (jcalcote) [inline install.sh]: Why patch vsm-controller in place? Why not just change the source version in the vsm project so it's deployed with the correct code?

**Pull request merged**